includes/definitions.inc.php

```diff
@@ -520,6 +520,26 @@
 $config['os'][$os]['over'][0]['text']  = 'Device Traffic';
 $config['os'][$os]['over'][1]['graph'] = 'device_processor';
 $config['os'][$os]['over'][1]['text']  = 'CPU Usage';
+$config['os'][$os]['poller_modules']['aruba-controller']       = 0;
+$config['os'][$os]['poller_modules']['bgp-peers']              = 0;
+$config['os'][$os]['poller_modules']['cisco-ace-serverfarms']  = 0;
+$config['os'][$os]['poller_modules']['cisco-ace-loadbalancer'] = 0;
+$config['os'][$os]['poller_modules']['cisco-cbqos']            = 0;
+$config['os'][$os]['poller_modules']['cisco-cef']              = 0;
+$config['os'][$os]['poller_modules']['cisco-mac-accounting']   = 0;
+$config['os'][$os]['poller_modules']['cisco-ntp']              = 0;
+$config['os'][$os]['poller_modules']['cisco-voice']            = 0;
+$config['os'][$os]['poller_modules']['hr-mib']                 = 0;
+$config['os'][$os]['poller_modules']['ipmi']                   = 0;
+$config['os'][$os]['poller_modules']['ipSystemStats']          = 0;
+$config['os'][$os]['poller_modules']['junose-atm-vp']          = 0;
+$config['os'][$os]['poller_modules']['netscaler-vsvr']         = 0;
+$config['os'][$os]['poller_modules']['ntp']                    = 0;
+$config['os'][$os]['poller_modules']['ospf']                   = 0;
```

Comment:
I believe the asa's can run ospf as well, again does our code for this not pick it up?